RecoTracker/MkFit/plugins/MkFitOutputConverter.cc

```diff
@@ -341,6 +341,8 @@ TrackCandidateCollection MkFitOutputConverter::convertCandidates(const MkFitOutp
     const auto seedIndex = cand.label();
     LogTrace("MkFitOutputConverter") << " from seed " << seedIndex << " seed hits";
 
+    if (mkFitOutput.propagatedToFirstLayer())
```

Comment:
perhaps add something like ```rescale, to be consistent with TransientInitialStateEstimator::innerState used in CkfTrackCandidateMakerBase```

After staring at this more, why do we have this rescaling conditional here on propagation to first layer?

... also, in view of the discussion of this rescaling possibly be moved to the final fit, it may be practical to make this configurable as well.

Reply:
> perhaps add something like `rescale, to be consistent with TransientInitialStateEstimator::innerState used in CkfTrackCandidateMakerBase`

Comment is easy to add.

> 
> After staring at this more, why do we have this rescaling conditional here on propagation to first layer?

Here I followed your original suggestion. Not sure if there was a deeper reason.

> 
> ... also, in view of the discussion of this rescaling possibly be moved to the final fit, it may be practical to make this configurable as well.

Do you mean to make the rescaling factor configurable, or the rescaling itself?

Reply:
> Do you mean to make the rescaling factor configurable, or the rescaling itself?

I was thinking of just a flag to rescale or now.

can be both though.

Reply:
I believe that the reason for rescaling it only if propagating to the first layer is that otherwise backwardFit is run, and there rescaling is already done.

Reply:
There's now a configurable bool to rescale the error, or not.
Comments explain why this is done.